test/unit/org/apache/cassandra/batchlog/BatchlogEndpointFilterTest.java

```diff
@@ -45,6 +45,23 @@ public static void beforeClass()
         DatabaseDescriptor.setConfig(new Config());
     }
 
+    @Test
+    public void shouldSelect2HostsFromLocalRackIfPreferred() throws UnknownHostException
```

Comment:
IIUC the expected result contains *one* host from the local rack and *one* host from another non-local rack. From the method name it seems like both hosts should be selected from the local rack. Maybe we can rename the method accordingly?

Reply:
Yep good point.